config/client.go

```diff
@@ -93,6 +93,9 @@ type TiKVClient struct {
 	// MaxConcurrencyRequestLimit is the max concurrency number of request to be sent the tikv
 	// 0 means auto adjust by feedback.
 	MaxConcurrencyRequestLimit int64 `toml:"max-concurrency-request-limit" json:"max-concurrency-request-limit"`
+	// EnableReplicaSelectorV2 indicate whether to use the new replica-selector-v2.
+	// TODO(crazycs520): remove this config after the new replica-selector-v2 is stable.
```

Comment:
This TODO comment is removed. Do we still plan to remove this configuration when v2 is stable?